unit-tests/test-coldlaunch.js

```diff
@@ -112,15 +112,14 @@ describe('cold-launch', () => {
 
     // --app not provided throws an error
     it('error thrown when --app not provided', () => {
-
       let errorFunc = () => {
+
         let options = {
           'metrics': 'metrics.ldjson',
           'runs': 30
         };
         let coldlaunch = new ColdLaunch(options);
       }
-
```

Comment:
haha alright my bad i'm confused, so in a different place above you said put a blank line after the let: let errorFunc = () => {

So I have a blank line at 116 above to follow that rule... but you want that removed here?